pkg/ui/workspaces/cluster-ui/src/timeScaleDropdown/timeFrameControls.tsx

```diff
@@ -84,8 +83,8 @@ export const TimeFrameControls = ({
       >
         <Button
           onClick={handleChangeArrow(ArrowDirection.CENTER)}
-          disabled={center}
-          className={cx("_action", center ? "disabled" : "active", "btn__now")}
+          disabled={false}
```

Comment:
I think we can just remove this prop entirely.

Reply:
Done